README.md

```diff
@@ -1 +1,9 @@
-# amrita_questionpapers
\ No newline at end of file
+# amrita_questionpapers
+
+## Contributing
+
+We welcome contributions! Please see our [Contributing Guide](CONTRIBUTING.md) for more details.
+
+## Code of Conduct
```

Comment:
Comment out the Code of Conduct part as it's not up yet. Don't remove it, just comment it out.